app/generator/content/part2/get-table/map-funding-rows.js

```diff
@@ -1,15 +1,16 @@
 const toCurrencyString = require('../../../to-currency-string')
-const mapReductionIds = require('./map-reduction-ids')
+const orderFunding = require('./order-funding')
```

Comment:
There was no ordering on funding previously, this caused the table rows to jump around on each generation.  Adding order so consistent and logical flow.